app/controllers/api/events_controller.rb

```diff
@@ -2,8 +2,6 @@ module Api
 
   class EventsController < ApplicationController
 
-    skip_before_action :verify_authenticity_token, if: :json_request?
```

Comment:
i assume that since we don't raise auth token error anymore this won't be a problem.